drivers/SPI.cpp

```diff
@@ -77,7 +77,7 @@ SPI* SPI::_owner = NULL;
 SingletonPtr<PlatformMutex> SPI::_mutex;
 
 // ignore the fact there are multiple physical spis, and always update if it wasnt us last
-void SPI::aquire() {
+void SPI::acquire() {
```

Comment:
This is protected method, should not be just renamed, but the correct one added instead for backward compatibility. 
Please remove this change from here, should be sent separately.